src/components v2/Message/Message.spec.js

```diff
@@ -0,0 +1,70 @@
+import Message from '@/components/Message/Message.vue';
+import { createWrapper } from '@/utils/unitTest';
+
+describe('Message.vue', () => {
```

Comment:
**nit-pick(non-blocking): ** can we name this `Components v2 / Message.vue` to differentiate the tests from the v1 components?